doc/api/process.md

```diff
@@ -1890,6 +1898,20 @@ The `process.version` property returns the Node.js version string.
 console.log(`Version: ${process.version}`);
 ```
 
+## process.computedVersion
+<!-- YAML
+added: REPLACEME
+-->
+
+* {number}
+
+The `process.computedVersion` property is the result of
+`(NODE_MAJOR_VERSION << 16) + (NODE_MINOR_VERSION << 8) + NODE_PATCH_VERSION`
+
+This property is useful for inline version checks. Using the process above a
+number such as `591872` (for version `9.8.0`) may be selected and easily used
+for a version check (`process.computedVersion >= 591872`).
```

Comment:
Will the fact that the computed version does not incorporate the pre-release tag impact anything here? For instance, every version built on master will have the same computed version until the next major release.

Reply:
i see at least two ways to solve this if its judged to be a problem
a) `process.computedVersion >= 591872 && process.prerelaseTag`
b) reserve a bit in `computedVersion` for prerelease, however i'm not sure how to do that without breaking semver comparisons

Reply:
Yeah, as long as folks know to check both. Perhaps an additional note in the docs?